extension/intents/search/search.js

```diff
@@ -391,9 +391,25 @@ intentRunner.registerIntent({
       });
 
       await focusSearchTab();
-
       await content.lazyInject(tabId, "/intents/search/queryScript.js");
-      const searchInfo = await callScript({ type: "searchResultInfo" });
+
+      const engines = await browser.search.get();
+
+      const searchInfo = await callScript({
+        type: "searchResultInfo",
+        searchEngine: engines.find(engine => engine.isDefault).name,
+      });
```

Comment:
I don't really know how expensive it is to get the search engine list each time, but I would worry about doing this. I think there's an easier test that can be done above

Reply:
Since you requested I use `location.origin` to retrieve the search engine being used, the `searchEngine` option to `callScript` became irrelevant and has been removed.